libs/libavro/Makefile

```diff
@@ -0,0 +1,55 @@
+#
+# libavro - Makefile for Apache AvroC library
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=avro-c
```

Comment:
The Apache Avro project has a C version named "Avro C"
https://avro.apache.org/docs/current/api/c/

When you build this project, it emits the library: libavro.so.23.0.0

Apache has no project known as "libavro"

What do you recommend I rename the bins?  I left Apache's naming scheme alone as to not confuse anyone.